frontend/packages/ui/src/features/wallet-new/hooks/use-wallet.ts

```diff
@@ -0,0 +1,19 @@
+import { useAccount } from '@gear-js/react-hooks';
+import { useState } from 'react';
+import { WALLET } from '../consts';
+import { WalletId } from '../types';
+
+function useWallet() {
```

Comment:
Let's reuse `useWallet` hook from original implementation. No need to repeat the logic.

This applies to shared `assets` as well.